core/app/helpers/spree/taxons_helper.rb

```diff
@@ -7,9 +7,9 @@ def taxon_preview(taxon, max = 4)
       products = taxon.active_products.select("DISTINCT (spree_products.id), spree_products.*, spree_products_taxons.position").limit(max)
       if products.size < max
         products_arel = Spree::Product.arel_table
-        taxon.descendants.each do |taxon|
+        taxon.descendants.each do |t|
```

Comment:
I think this is the same one as above: `t` is much less descriptive than `taxon`. If the issue is variable name shadowing, maybe rather one like `descendent_taxon`?